types/test/plugin-test.ts

```diff
@@ -14,7 +14,13 @@ const plugin: PluginObject<Option> = {
   }
 }
 const installer: PluginFunction<Option> = function(Vue, option) { }
+function NoOptions( _vue: typeof Vue ){};
```

Comment:
The typeof shouldn't be necessary, also it's better to use an uppercase version like `_Vue` or `Vue`

Reply:
Since is defined as `PluginFunction<T> = (Vue: typeof _Vue, options?: T) => void;` the compiler will complain on a direct usage of Vue as a type.